### PR TITLE
16: Move remaining tests

### PR DESCRIPTION
**Pull request by rehammuzzamil**
https://github.com/OpenSRP/opensrp-plan-evaluator/issues/16

### CODE REVIEW

File: src/main/java/org/smartregister/domain/Form.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package org.smartregister.domain;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+public class Form {
```
Comment (githengi) [inline src/main/java/org/smartregister/domain/Form.java:10]: This class should remain in core its not being used on this library

File: src/main/java/org/smartregister/domain/FormField.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
+package org.smartregister.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+public class FormField {
```
Comment (githengi) [inline src/main/java/org/smartregister/domain/FormField.java:8]: This class should remain in core its not being used on this library

File: src/main/java/org/smartregister/domain/SubFormDefinition.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
+package org.smartregister.domain;
+
+import java.util.List;
+
+public class SubFormDefinition {
```
Comment (githengi) [inline src/main/java/org/smartregister/domain/SubFormDefinition.java:5]: This class should remain in core its not being used on this library

File: src/main/java/org/smartregister/domain/FormField.java (hunk 2 above)
Comment (rehammuzzamil) [reply src/main/java/org/smartregister/domain/FormField.java:8]: Actually FormTest class is being referenced in ContactPointTest and UserTest class, therefore I had to move these classes too.

Comment (rehammuzzamil) [reply src/main/java/org/smartregister/domain/FormField.java]: Moved back to core project

File: src/main/java/org/smartregister/domain/SubFormDefinition.java (hunk 3 above)
Comment (rehammuzzamil) [reply src/main/java/org/smartregister/domain/SubFormDefinition.java]: Moved back to core project

**Comment by rehammuzzamil** [comment]
> Bump up version

Done

**Pull request merged**